app/views/issues/_send_to_menu.html.erb

```diff
@@ -1,16 +1,22 @@
-<% sync_plugins = Dradis::Plugins::with_feature(:issue_sync) %>
-<% if sync_plugins.any? %>
 <span class="dropdown">
   <a class="dropdown-toggle action-link" data-name="issues" data-toggle="dropdown" href="javascript:void(0)">
     Send to...
   </a>
   <ul class="dropdown-menu" role="menu">
-  <% Dradis::Plugins::with_feature(:issue_sync).each do |plugin| %>
-    <%=
-      plugin_path = ActiveSupport::Inflector.underscore(ActiveSupport::Inflector.deconstantize(plugin.name))
-      render partial: "#{plugin_path}/issues/send_to_menu"
-    %>
+  <% sync_plugins = Dradis::Plugins::with_feature(:issue_sync) %>
+  <% if sync_plugins.any? %>
+    <% sync_plugins.each do |plugin| %>
+      <%=
+        plugin_path = ActiveSupport::Inflector.underscore(ActiveSupport::Inflector.deconstantize(plugin.name))
+        render partial: "#{plugin_path}/issues/send_to_menu"
+      %>
+    <% end %>
+  <% end %>
+  <% unless defined?(Dradis::Pro) && sync_plugins.include?(Dradis::Pro::Issuelib::Engine) %>
```

Comment:
@etdsoft This returns an error `NameError: uninitialized constant Dradis::Pro::Issuelib` if the IssueLibrary is not installed. It might be good to stringify the`sync_plugins` list and use the string `"Dradis::Pro::Issuelib::Engine"` to check for inclusion instead.